.circleci/config.yml

```diff
@@ -228,21 +237,52 @@ workflows:
   version: 2
   grakn-core-ci:
     jobs:
-      - build
-      - client-java
-      - client-nodejs
-      - client-python
-      - common
-      - console
-      - graql
-      - server
-      - workbase
-      - test-integration
-      - test-integration-reasoner
-      - test-integration-reasoner-general
-      - test-integration-reasoner-components
-      - test-integration-analytics
-      - test-end-to-end
+      - lint
```

Comment:
@lolski this is a valid point! it's not _required_ per se (because `bazel test` will build everything it needs), but it's better to know that everything builds correctly

Reply:
@lolski actually, let's do it until we have remote cache or remote execution. Using `persist_to_workspace` to save bazel cache **not** to be a good strategy because _uploading_ the cache takes > 5 minutes, see [this run](https://circleci.com/gh/graknlabs/grakn/9620)